src/internal/connector/exchange/attachment.go

```diff
@@ -55,19 +60,27 @@ func uploadAttachment(
 
 	// item Attachments to be skipped until the completion of Issue #2353
 	if attachmentType == models.ITEM_ATTACHMENTTYPE {
-		name := ""
-		if attachment.GetName() != nil {
-			name = *attachment.GetName()
+		prev := attachment
+		attachment, err = support.ToItemAttachment(attachment)
+
+		if err != nil {
+			// TODO: Update to support PII protection
+			name := ""
+			if prev.GetName() != nil {
+				name = *prev.GetName()
+			}
+
+			logger.Ctx(ctx).Infow("item attachment uploads are not supported ",
+				"attachment_name", name,
+				"attachment_type", attachmentType,
+				"internal_item_type", getItemAttachmentItemType(prev),
+				"attachment_id", *prev.GetId(),
+			)
+
+			fmt.Println("Error returned: " + err.Error())
```

Comment:
Debugging statement?  Seems like it could be removed.

Reply:
Resolved within [ba6118d](https://github.com/alcionai/corso/pull/2355/commits/ba6118d34c7bf97dda9dea0bd92b9663b1f9820b)